MovieDemoApp/Views/Movie Detail/Cast And Crew/CastAndCrewView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CastAndCrewView.swift
+//  MovieDemoApp
+//  Copyright © 2020 Deloitte Digital. All rights reserved.
+//
+
+import SwiftUI
+
+struct CastAndCrewView: View {
+    let castAndCrews: [CastAndCrew]
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("Cast and Crew")
+                .padding(.all)
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(alignment: .top, spacing: 10) {
+                    ForEach(castAndCrews) { castAndCrew in
```

Comment:
Can we use map here?

Reply:
I tried using :

    `castAndCrews.forEach( CastAndCrewSingleView(castAndCrew: $0 ))`

But I am getting error: 
> Static member 'leading' cannot be used on instance of type 'HorizontalAlignment'

Reply:
you can't use forEach inside body

Reply:
you can't use .foreach inside body
ForEach is a swift UI keyword.

Reply:
@NilaakashSingh so this is specific to Swift UI and we cannot use .map types of functions here?

Reply:
In Apple Doc: 

ForEach is an struct:

    `/// A structure that computes views on demand from an underlying collection of
     /// of identified data.
     @available(iOS 13.0, OSX 10.15, tvOS 13.0, watchOS 6.0, *)
     public struct ForEach<Data, ID, Content> where Data : RandomAccessCollection, ID : 
     Hashable {

        /// The collection of underlying identified data.
        public var data: Data

        /// A function that can be used to generate content on demand given
        /// underlying data.
        public var content: (Data.Element) -> Content
    }`

and .forEach is closure: 

    /// - Parameter body: A closure that takes an element of the sequence as a
    ///   parameter.
    @inlinable public func forEach(_ body: (Self.Element) throws -> Void) rethrows

Reply:
yeah @swapnil-salunke we can't use .map directly with UI components of SwiftUI. On the other hand, we can do it via the Combine model. We can keep it as it is now.